docs/make_external_gallery.py

```diff
@@ -0,0 +1,156 @@
+        description="PVGeo's example gallery",
+        link="https://pvgeo.org/examples/index.html",
+        image="pvgeo.png"),
+    tetgen=Example(title="TetGen's example gallery",
+        description="TetGen's example gallery",
+        link="http://tetgen.pyvista.org/examples/index.html",
+        image="tetgen.png"),
+    mesh_fix=Example(title="PyMeshFix's example gallery",
+        description="PyMeshFix's example gallery",
+        link="http://pymeshfix.pyvista.org/examples/index.html",
+        image="pymeshfix.png"),
+    orvisu=Example(title="Orvisu Demo Application",
+        description="Orvisu Demo Application",
+        link="https://github.com/BartheG/Orvisu",
+        image="orvisu.gif"),
+    flem=Example(title="FLEM: A diffusive landscape evolution model",
+        description="FLEM: A diffusive landscape evolution model",
+        link="https://github.com/johnjarmitage/flem",
+        image="flem.png"),
+    optimization=Example(title="Optimization visualization with PyVista",
+        description="Optimization visualization with PyVista",
+        link="https://gist.github.com/hichamjanati/6668d91848283c31ac18d801552fb582",
+        image="optimization.gif"),
+    anvil_cirrus_plumes=Example(title="Anvil Cirrus Plumes",
+        description="Dr. Morgan O'Neill at Stanford University is researching Above Anvil Cirrus Plumes and its dynamics as a hydraulic jump.",
+        link="https://www.youtube.com/watch?v=cCPjnF_vHxw&feature=youtu.be",
+        image="anvil_cirrus_plumes.png"),
+    damavand=Example(title="Damavand Volcano",
+        description="Visualize 3D models of Damavand Volcano, Alborz, Iran.",
+        link="https://nbviewer.jupyter.org/github/banesullivan/damavand-volcano/blob/master/Damavand_Volcano.ipynb",
+        image="damavand_volcano.gif"),
+    atmos_conv=Example(title="Atmospheric Convection",
+        description="Atmospheric convection plays a key role in the climate of tidally-locked terrestrial exoplanets: insights from high-resolution simulations",
+        link="https://dennissergeev.github.io/exoconvection-apj-2020/",
+        image="atmospheric_convection.jpeg"),
+    # entry=Example(title="",
+    #     description="",
+    #     link="",
+    #     image=""),
+)
+
+
+###############################################################################
+
+def make_example_gallery():
+    filename = "./external_examples.rst"
+    if os.path.exists(filename):
+        os.remove(filename)
+    with open(filename, "w") as f:
+        f.write("""
+External Examples
+=================
+
+Here are a list of longer, more technical examples of what PyVista can do!
+
+.. caution::
+
+    Please note that these examples link to external websites.
+    If any of these links are broken, please raise an issue on the repository.
+
+
+Do you have a technical processing workflow or visualization routine you
+would like to share?
+If so, please consider sharing your work here by clicking the `Edit on GitHub`
```

Comment:
We've removed the "Edit on GitHub" button for the time being.  Perhaps add "email us"?